python/Ganga/GPIDev/Lib/Tasks/CoreUnit.py

```diff
@@ -32,7 +33,7 @@ def createNewJob(self):
             if (f == "postprocessors" and len(getattr(self, f).process_objects) > 0):
                 j.postprocessors = copy.deepcopy(addProxy(self).postprocessors)
             elif (f != "postprocessors" and getattr(self, f)):
-                setattr(j, f, copy.deepcopy(getattr(self, f)))
+                setattr(stripProxy(j), f, copy.deepcopy(getattr(self, f)))
```

Comment:
I think that `j` should be a raw job here? `makeRegisteredJob()` certainly returns one.